### ISSUE TITLE
"breadcrumb" bar should display what homework for TA grading

### ISSUE THREAD

**Issue by bmcutler** [Submitty/Submitty#1505]
Doesn't seem to have new items added when on HW grading page.  And check the other pages too.  Make sure the breadcrumb bar shows appropriate details on the various pages in the system.

**Comment by bmcutler** [comment]
from the homepage, when you click on "grade" for HW1, the breadcrumb should be
csci1000 > HW1 Grading    (shows the grading status/statistics)

Then when you click on "grading details", the breadcrumb should be
csci1000 > HW1 Grading > Student Index   (shows the list of all students)

Then when you click on a specific student to grade, the breadcrumb should be
csci1000 > HW1 Grading > Student Index > smithj   (shows the rubric etc for this student)


When you click on item from the instructor bar (black bar at the top), the breadcrumb bar should be

csci1000 > Excused Absence Extensions       for each item in the bar.
Remove "Admin" (irrelevant detail to typical instructor user)
Use the long name for each page,  (e.g. clicking on "Students" takes  you to "View Students", use the latter in the breadcrumb bar)

**Comment by bmcutler** [comment]
The breadcrumb bar for the TA grading pages looks good, but the links within the bar don't appear to be connect to URLs.

**Comment by tcyrus** [comment]
I thought I tested the links. I'll check it again and fix it.

**Comment by tcyrus** [comment]
I found the issue which I think you are referring to. On the TA grading view, the links are valid, but cannot be selected because the header is covered up (in terms of it's `z-index`). This issue looks like it was introduced as part of #1564.

**Comment by bmcutler** [comment]
I've installed both PRs, but this still isn't working on Mac Chrome.

**Comment by andrewaikens87** [comment]
I just tested on Mac Chrome and it seems to be working (the breadcrumbs and the panels staying over the header). We should discuss more on Wednesday. The one thing I did notice while testing was that if the reset button is clicked, the `z-index` of the panels are reset to zero (and then are hidden by the header). I checked the ta-grading.css file and it seems that the `z-index` for the panels and the grading_toolbar were incorrect in #1582 (based on the values that are assigned in the ta-grading.js file). This fix will be included in the draggable-toolbar branch once I test on Mac.
